cmd/diff.go

```diff
@@ -52,21 +52,20 @@ var diffCmd = &cobra.Command{
 
 func checkDiffArgNum(args []string) error {
 	if len(args) != 2 {
-		return errors.New("'diff' requires two images as arguments: container diff [image1] [image2]")
+		return errors.New("'diff' requires two images as arguments: container-diff diff [image1] [image2]")
 	}
 	return nil
 }
 
 func diffImages(image1Arg, image2Arg string, diffArgs []string) error {
 	diffTypes, err := differs.GetAnalyzers(diffArgs)
 	if err != nil {
-		glog.Error(err.Error())
-		return errors.New("Could not perform image diff")
+		return fmt.Errorf("err msg: %s", err.Error())
```

Comment:
just return err or make this more informative?

Reply:
fixed up the errors in this file